client/pages/inbox.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+class Inbox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      message: []
+    };
+  }
+
+  componentDidMount() {
+    const userId = window.localStorage.getItem('userId');
+    fetch(`/api/messages/${userId}`)
```

Comment:
You can change your server to only expect `/api/messages` with no user id at the end. the token should be sent in the headers

```js
fetch('/api/messages', {
  headers: {
    'X-Access-Token': token
  }
})
```